src/main/java/com/ays/backend/user/security/JwtUserDetails.java

```diff
@@ -57,4 +55,12 @@ public boolean isCredentialsNonExpired() {
     public boolean isEnabled() {
         return true;
     }
+
+    public long getId() {
```

Comment:
It might be healthier to use objects instead of primitives.

Reply:
I changed it as a wrapper class (long -> Long)